test/filters.js

```diff
@@ -707,6 +713,93 @@ describe('filters', function () {
     })
   })
 
+  describe("toCurrency", function(){
+    let errorMessage = "invalid currency value or type";
+
+    it('should return {value: 8000, type: "INR"}"', () => {
+      const dst = { value: 8000, type: "INR" };
+      return test('{{ 8000 | toCurrency: "INR" }}', JSON.stringify(dst));
+    });
+
+    it('should return {value: 100, type: "USD"} for params 100 and "USD"', () => {
+      const dst = { value: 100, type: "USD" };
+      return test('{{ 100 | toCurrency: "USD" }}', JSON.stringify(dst));
+    });
+
+    it('should return {value: 100, type: "USD"} for params 100 and "USD"', () => {
+      const dst = { value: 150.1, type: "USD" };
+      return test('{{ 150.1 | toCurrency: "USD" }}', JSON.stringify(dst));
+    });
+
+    it("should throw error when currency value is 0", () =>
```

Comment:
Why are we not allowing them to create `0 USD`? isn't it a valid currency?

Reply:
Initially I felt that 0 {{currency}} does not make sense because contracts will involve money. But after reading the review comment, I do understand that there can be usecases where user may end up saying that balance is 0 {{currency}} or charge for breaking the contract will be 0 {{currency}} and such. Makes sense. Have removed this condition.